tests/queries/0_stateless/02013_json_function_null_column.sql

```diff
@@ -0,0 +1,10 @@
+-- Tags: no-fasttest
+
+drop table if exists test_table;
+
+create table test_table (col String, col_nullable Nullable(String)) engine MergeTree order by col;
+insert into test_table select '{"string_value":null}' as col, '{"string_value":null}' as col_nullable;
+
+select JSONExtract(col, 'string_value', 'Nullable(String)') as res1, JSONExtract(col_nullable, 'string_value', 'Nullable(String)') as res2, JSONExtract(assumeNotNull(col_nullable), 'string_value', 'Nullable(String)') as res3 from test_table;
+
+drop table test_table;
```

Comment:
The test looks too complex for this simple case. Can we use something more simple here, without creating tables, maybe like [my test](https://github.com/vitlibar/ClickHouse/blob/f0bf01212746c414acb00516cdb240c423819058/tests/queries/0_stateless/02023_json_extract_nullable_argument.sql) for example? It could be run as a fast test then.

Reply:
Done.

Reply:
It appears it can't be run as a fast test anyway because the `fasttest` build is compiled without libraries, i.e. without rapidjson and without simdjson.

Reply:
I thought your tests might work "magically")